tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/intersection/IntersectionTypeTest.java

```diff
@@ -63,6 +63,11 @@ public void testRuntimeTypeNameOfIntersectionType() {
         BRunUtil.invoke(readOnlyIntersectionResults, "testRuntimeTypeNameOfIntersectionType");
     }
 
+    @Test
```

Comment:
This isn't the correct file for this test IMO. We can have it in type test tests.